docs/content/concepts/apis/exporting-apis.md

```diff
@@ -303,7 +311,10 @@ When reconciling exported APIs, controllers usually interact with the API resour
 
 ### Endpoint Slices
 
-An API provider that implements a controller should also create something called an `APIExportEndpointSlice` alongside their `APIExport`. This will "slice and dice" the list of available virtual workspace endpoints. For example, a [Partition](../sharding/partitions.md) can be provided to restrict provided virtual workspace URLs to a part of the sharded kcp setup.
+An API provider that implements a controller can optionally also create something called an `APIExportEndpointSlice` alongside their `APIExport`. This will "slice and dice" the list of available virtual workspace endpoints. For example, a [Partition](../sharding/partitions.md) can be provided to restrict provided virtual workspace URLs to a part of the sharded kcp setup.
+
+!!! information "Changes with kcp v0.28"
+    Since kcp 0.28, kcp creates an `APIExportEndpointSlice` per `APIExport` automatically. So unless a certain partition should be targetted, it is not required to create one manually. If you wish to disable this behaviour, label your `APIExport` with `apiexports.apis.kcp.io/skip-endpointslice=true`.
```

Comment:
I _think_ it should be `targeted`
```suggestion
    Since kcp 0.28, kcp creates an `APIExportEndpointSlice` per `APIExport` automatically. So unless a certain partition should be targeted, it is not required to create one manually. If you wish to disable this behaviour, label your `APIExport` with `apiexports.apis.kcp.io/skip-endpointslice=true`.
```